### PR TITLE
add new zh-CN folder for intro-to-storybook/vue and translate get-sta…

### PR DESCRIPTION
**Pull request by zydmayday**
First commit to init intro-to-storybook/vue for zh-CN.

- [x] get-started.md 2021-02-01
- [x] simple-component.md 2021-02-04
- [x] composite-component.md 2021-02-08
- [x] data.md 2021-02-18
- [x] screen.md 2021-02-20
- [x] deploy.md 2021-02-24
- [x] test.md 2021-02-27
- [x] using-addons.md 2021-03-04
- [x] conclusion.md 2021-03-08
- [x] contribute.md 2021-03-12

### CODE REVIEW

**Comment by jonniebigodes** [comment]
@zydmayday this is fantastic, thank you very much for the time and effort you're putting into the translation. Really stoked to see it done and get the tutorial available to more and more languages. I saw that you just started and we're in a good place for one thing. If you don't mind update the get started based on this pull request #435 . The intention here is to move to a template approach and allow you and the readers have a more seamless experience. 

Let me know when you have it all translated so that we can get a proofreader and get it merged. 

Stay safe

**Comment by zydmayday** [comment]
Hi @jonniebigodes, really excited to hear your response. 
> If you don't mind update the get started based on this pull request #435
I am working on this once I finished I will push my code.

Have a nice day!

**Comment by jonniebigodes** [comment]
@zydmayday wanted to give you a heads up. The pull request i've mentioned was just merged. I'll leave it to you and ping me when you get the full tutorial adjusted so that we can find a proofreader and get the pull request merged.

Stay safe

**Comment by zydmayday** [comment]
Hi @jonniebigodes ,
https://storybook.js.org/tutorials/intro-to-storybook/vue/en/simple-component/
I found in this chapter, the link `previous chapter` didn't work properly, could you check it?

**Comment by jonniebigodes** [comment]
@zydmayday sorry about that. With a recent change in how we're handling the tutorials and official documentation some work was required and led to some impromptu mishandling of the links within the content. I just merged #453 which addressed the issue. What i would like from you is to update your fork so that the changes introduced are rolled in and also please let me know if there's any issues with the links.

Stay safe

**Comment by jonniebigodes** [comment]
@zydmayday if you could adjust your pull request and factor in the changes introduced by #456 i would really appreciate it, as there were a couple of issues that were factored in. Let me know once the changes are up and you wrapped up the translation so that we can reach out to the community for a proofreader and get this pull request merged and the translation released.

 Sounds good?

**Comment by zydmayday** [comment]
Hi @jonniebigodes！
Sorry for late! I have almost finished my work on it, so please kindly have a review on it  with the proofreaders' help.
I will response and update my code ASAP if change necessary.

**Comment by zydmayday** [comment]
Should I also mension some information in issue https://github.com/chromaui/learnstorybook.com/issues/3?

**Comment by jonniebigodes** [comment]
@zydmayday really stoked that you already completed the translation. I'll reach out to the community and see if someone is willing to proofread it. Once this is proofread and merged I'll update the issue (#3)  with your GitHub username as the primary translator for the Vue edition of the tutorial.

**Comment by jonniebigodes** [comment]
@zydmayday just wanted to give a heads up. I've reached out to the community in hope of getting someone to proofreader. If no one comes forward in a couple of weeks I'll merge your pull request and we proceed from there. Sounds good?

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@zydmayday just a couple of items that I would need for you to take care of from our end. I'll make another push in the community for a proofreader, if no one comes around we'll merge it and address improvements with future pull requests.

Let me know once you've taken care of the items I mentioned!

Sounds good?


Stay safe

**Comment by zydmayday** [comment]
Hi @jonniebigodes, Thanks for your feedback.

I modified the below things. 

- change comment in code into English
- modify diff block, add `+` mark
- modify some text according to the change of original english version
- remove english text

Kindly have a check.

Stay safe.

File: content/intro-to-storybook/vue/zh-CN/get-started.md
Comment (jonniebigodes) [inline content/intro-to-storybook/vue/zh-CN/get-started.md:25]: @zydmayday could you revert the comments in the snippets for this page to their native form as well?

File: content/intro-to-storybook/vue/zh-CN/simple-component.md
Comment (jonniebigodes) [inline content/intro-to-storybook/vue/zh-CN/simple-component.md:158]: @zydmayday if you don't mind adding the missing "+" here, in this case where adding the global styles to Storybook.

File: content/intro-to-storybook/vue/zh-CN/composite-component.md
Comment (jonniebigodes) [inline content/intro-to-storybook/vue/zh-CN/composite-component.md:240]: @zydmayday it seems that this needs a bit of work. I was checking the preview associated with your pull request and this link wasn't rendering correctly. It's actually showing the full text.

File: content/intro-to-storybook/vue/zh-CN/deploy.md
Comment (jonniebigodes) [inline content/intro-to-storybook/vue/zh-CN/deploy.md:20]: @zydmayday could you do another pass at this section? It seems that this section or some portions of it are a bit outdated! Compare it with the English version and adjust accordingly if you don't mind.

File: content/intro-to-storybook/vue/zh-CN/using-addons.md
Comment (jonniebigodes) [inline content/intro-to-storybook/vue/zh-CN/using-addons.md:85]: @zydmayday it seems that you might have missed a "[" here and this prevents the link to be properly rendered and shown.

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@zydmayday thank you for taking care of the feedback I've mentioned. I was taking another look before we could move into getting a proofreader from the community and get the pull request merged and I've stumbled into a couple of issues that require some attention.

Let me know once you've addressed them and I'll make another push and ask in our community for a proofreader and see if anyone is willing to help us!

As I've mentioned above if no one comes forward I'll merge this and we'll address the improvements with further pull requests. Sounds good?

Stay safe

File: content/intro-to-storybook/vue/zh-CN/composite-component.md
Comment (zydmayday) [reply content/intro-to-storybook/vue/zh-CN/composite-component.md]: I fixed the link issue.

I am not sure if it is deliberately just use the left parenthesis in the last line of composite-component.md.  
refer here: https://github.com/chromaui/learnstorybook.com/blob/master/content/intro-to-storybook/vue/en/composite-component.md

I added a right parenthesis also.

**Comment by jonniebigodes** [comment]
@zydmayday truly sorry for not getting this merged sooner. I asked around a couple of community members for a proofreader and none step forward. With that, this pull request has been on hold for more than enough time. I'm going to merge this. One final thing I need from you if you don't mind. Could you jump onto our [Discord Server](https://discord.gg/UUt2PJb) and message me (same username) so that I can follow up with you on one small item?

Really appreciate the time and effort you've put into this.

Stay safe

**Pull request merged**
